src/main/java/com/google/sps/data/Meal.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+public class Meal {
+    private final String title;
+    private final String description;
+    private final ArrayList<String> ingredients;
+    private final String type;
```

Comment:
If you expect `type` to only take a fixed set of options (e.g. you'll eventually define ~20 possible values, like "Italian", "Chinese", etc.), consider using an [Enum](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) instead of a String.

Same applies to ingredients - if we plan to support some limited set of them, consider representing them as Enums.

Reply:
Actually, I do not know yet, because in my opinion it really depends on dataset we will have. It will be wonderful to have this information loaded automatically. If we succeed, then I am not sure we will be able to limit them. 
However, if we need to add this information manually, I think it will be limited.

Reply:
I think that even if we load the information automatically, we should still try to run in through some kinda preprocessor to limit / correct / validate the data -> an `Enum` may still be a viable option.

Having said that, i agree that it's too early to tell so we can leave it as it is and revisit later.